src/client.js

```diff
@@ -54,7 +56,7 @@ export class Saturn {
       this._monitorPerformanceBuffer()
     }
     this.storage = this.opts.storage || memoryStorage()
-    this.loadNodesPromise = this._loadNodes(this.opts)
+    this.loadNodesPromise = this.opts.experimental ? this._loadNodes(this.opts) : null
```

Comment:
@DiegoRBaquero added this to only load nodes for the % we desire per a / b testing.